<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSearchSpotify</class>
 <widget class="QDialog" name="DlgSearchSpotify">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>463</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Spotify Search Music</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="leMusicSearch"/>
       </item>
       <item>
        <widget class="QPushButton" name="pbSearch">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Market:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leMarket"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="cbAlbum">
       <property name="text">
        <string>album</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbArtist">
       <property name="text">
        <string>artist</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbPlaylist">
       <property name="text">
        <string>Playlist</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="cbTrack">
       <property name="text">
        <string>Track</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbShow">
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbEpisode">
       <property name="text">
        <string>Episode</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="lwMusics">
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pbInsert">
       <property name="text">
        <string>Insert</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="teLogSearch"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
